README.md

```diff
@@ -31,6 +31,7 @@ Note that this is not intended to be a comprehensive set of recommendations. Rat
   * [MAX_PATH explanation and workarounds](windows-environment.md#max_path-explanation-and-workarounds)
   * [Compiling native Addon modules](windows-environment.md#compiling-native-addon-modules)
 * [Building for cross-platform](building-for-cross-platform.md)
+  * [NPM module to update Windows Registry](building-for-cross-platform.md#Reading and Writing to the Windows Registry in-process from Node.js)
```

Comment:
How about we label this `Windows services and settings` instead? That way we can expand beyond the registry.